generated/explanations/euler.md

```diff
@@ -0,0 +1,62 @@
+## Header
+This is the course header. This will be added on top of every page. Go to [DoDAO.io](https://www.dodao.io) to know more.
+
+ ---
+ 
+ ## Euler
+ 
+ **About Euler**        
+
+### What is Euler 
+Euler is a non-custodial DeFi protocol that allows users to lend and borrow and the main factor that differentiates Euler form other trusted protocols like Aave and Compound is that,  in Euler any asset that has a WETH pair on Uniswap v3 can be added as a lending market on Euler by anyone straight away. There is a  significant unmet demand for lending and borrowing  the long tail of crypto assets which are not listed on Aave or Compound. Euler is managed by holders of a protocol native governance token called Euler Governance Token (EUL). 
```

Comment:
Please use peppertype for this paragraph and other also as the sentence formation can be improved quite a bit.